core/src/MegaMolGraph.cpp

```diff
@@ -222,6 +222,17 @@ bool megamol::core::MegaMolGraph::SetParameter(std::string const& paramName, std
     return true;
 }
 
+bool megamol::core::MegaMolGraph::SetParameterHighlight(std::string const& paramName, bool is_highlight) {
+    auto param_slot_ptr = FindParameterSlot(paramName);
+    auto param_ptr = getParameterFromParamSlot(param_slot_ptr);
+
+    if (!param_ptr)
+        return false;
+    param_ptr->SetGUIHighlight(is_highlight);
+
+    return true;
+}
+
```

Comment:
I prefer we dont pollute the graph API with with hacks for unrelated features. The graph should support only a minimal set of operations. 

It is totally fine to simply get a parameter from the graph and ask it directly to do things, e.g:

https://github.com/UniStuttgart-VISUS/megamol/blob/master/frontend/services/lua_service_wrapper/Lua_Service_Wrapper.cpp#L567-L575

https://github.com/UniStuttgart-VISUS/megamol/blob/master/frontend/services/lua_service_wrapper/Lua_Service_Wrapper.cpp#L580-L587